soc/xtensa/esp32s2/soc.h

```diff
@@ -23,6 +23,10 @@
 
 void __esp_platform_start(void);
 
+static inline uint32_t esp_core_id(void)
```

Comment:
Is this really used? It is in `esp32s2` soc.

Reply:
yes, because interrupt allocator in esp32 and esp32s2 share the same code, so this function is needed here.